src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -2,29 +2,41 @@
 
 import nextstep.subway.applicaion.dto.LineRequest;
 import nextstep.subway.applicaion.dto.LineResponse;
-import nextstep.subway.domain.Line;
-import nextstep.subway.domain.LineRepository;
+import nextstep.subway.applicaion.dto.SectionRequest;
+import nextstep.subway.domain.*;
 import nextstep.subway.exception.DuplicatedException;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 
 import static java.util.stream.Collectors.*;
+import static nextstep.subway.common.consts.LineConsts.*;
 
 @Service
 @Transactional
 public class LineService {
     private LineRepository lineRepository;
+    private StationRepository stationRepository;
 
-    public LineService(LineRepository lineRepository) {
+    public LineService(LineRepository lineRepository, StationRepository stationRepository) {
         this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
     }
 
     public LineResponse saveLine(LineRequest request) {
         validateDuplicatedLine(request);
-        Line line = lineRepository.save(new Line(request.getName(), request.getColor()));
-        return new LineResponse(line);
+        Station upStation =
```

Comment:
```suggestion
        Station upStation = getStationById(request.getUpStationId());
```

특별한 이유가 없다면 줄구분을 할 필요는 없어 보이네요.
팀에서 정하기 나름이지만 보통 구글 자바 스타일 가이드를 많이 따르니 참고하시기 바랍니다.
협업 할 때 코드 스타일을 동일하게 가져가야 코드를 빠르게 파악하는데 도움이 됩니다.